htdocs/js/jquery.iconselector.js

```diff
@@ -311,4 +311,13 @@
         }
     };
 
+    var browseButton = $("#lj_userpicselect");
```

Comment:
This worked in my foundation branch, but in retrospect it's the only bit that I'm not 100% sure isn't broken when the deferred script tags are in the head. It should probably be outside this main function body, in a separate `jQuery( function() {...} );` callback, to make sure it's only called once the DOM is ready.